TrainingRNExpress/src/constants.js

```diff
@@ -5,6 +5,15 @@ export const ITEMS = [
   { name: 'Eat breakfast', checked: false }
 ];
 
+export const ROUTES = {
+  HOME: 'home',
+  BOOK: 'book'
+}
```

Comment:
Usually, we distinguish these constants by putting them into different files, but I think it's okay since this file is not too large